Python/context.c

```diff
@@ -1179,10 +1179,17 @@ static PyGetSetDef PyContextTokenType_getsetlist[] = {
     {NULL}
 };
 
+static PyMethodDef PyContextTokenType_methods[] = {
+    {"__class_getitem__",    (PyCFunction)Py_GenericAlias,  METH_O|METH_CLASS,
+    PyDoc_STR("A class-level getitem to make Token generic.")},
```

Comment:
Almost everywhere else the docstring is "See PEP 585".

Reply:
@1st1 requested me to change it (see above). I have no strong opinion either way.

Reply:
Hm. People aren’t going to understand what it says either way. “See PEP 585” at least gives a useful pointer, while wasting little space. @1st1

Reply:
Alright, sounds good, let's leave as is.